src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextFactoryTest.kt

```diff
@@ -207,7 +207,8 @@ class RecallNotificationContextFactoryTest {
       currentPrison = currentPrisonId,
       previousConvictionMainNameCategory = prevConsMainNameCategory,
       previousConvictionMainName = prevConsMainName,
-      inCustody = false
+      inCustody = false,
+      arrestIssues = false
```

Comment:
Or `null` perhaps?

Reply:
cant be null if inCustody is false when generating a recall notification. Only null if inCustody = true (then we never ask the question)